### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Antoine Aubry, Louis Haye, Zaïd Schouwey

### DISCUSSION THREAD

**Discussion by Artotrogo** [heig-vd-dai-course/heig-vd-dai-course#471] [Show and tell]
We want to create a game of Naval Battle :
- A small command line game.
- One player versus one player.
- The server will manage the matchmaking and will be the referee of the match.
- The board of the game will be displayed in the console and will be updated after each move.

**Comment by calystoxi** [comment]
Commit_id of the tag: 5d5ff17
Hash: 5d5ff17dcfe51e964311868db3876c315cbd81be
https://github.com/zaidschouwey98/DAI_naval_battle
